scripts/release.sh

```diff
@@ -314,10 +329,70 @@ main() {
     exit 1
   fi
 
-  # TODO: signing process
-  log_warning ""
-  log_warning "WARNING: The release has not been signed and published to github. This must be done manually."
-  log_warning ""
+  if [ "${DRY_RUN}" == "true" ] || [ "${NO_GH_RELEASE}" == 1 ]; then
+    log_warning ""
+    log_warning "WARNING: Skipping creating GitHub release, --no-gh-release is set."
+    log_warning "WARNING: If not running on DRY_MODE, please do the GitHub release manually."
+    log_warning ""
+  else
+    local gh_repo
+    local release_notes_temp_file
+    local release_url
+    local gh_release_args=()
+
+    case "${BRANCH}" in
+      "main")
+        gh_release_args=(--prerelease)
+        ;;
+      "release-3.5")
+        gh_release_args=(--latest)
+        ;;
+    esac
```

Comment:
3.4.x isn't supported yet?

Reply:
If the branch is:
* `main`: we want to mark it as a prerelease
* `release-3.5`: we want to mark it as latest
* `release-3.4`: we want to leave it as is (therefore, it doesn't need an additional argument)

Let me document it to ensure everything is clear.

Reply:
ack

Reply:
Done in 88d2d2472c32e4cfa0e7223fa0b173865f0ddf3c